main.py

```diff
@@ -70,6 +72,38 @@ def report(ctx, output):
     exporter.generate(project, 'report.html')
     pass
 
+@cli.command()
+@click.option('--version-file', default='version.csv', help='CSV Version file')
+@click.option('--overwrite', default=False, help='Overwrite database')
+@click.pass_context
+def import_csv(ctx, version_file, overwrite):
```

Comment:
This should be nammed "import"

Reply:
The word _import_ is reserved for python

Reply:
I'm talking about the name of the command

```
@click.command(name='import')
def import_file():
    pass
```